src/dotnet/APIView/ClientSPA/src/app/_helpers/common-helpers.ts

```diff
@@ -61,4 +61,8 @@ export function isDiffRow(row: CodePanelRowData) {
 
 export function convertRowOfTokensToString(rowOfTokens: StructuredToken[]): string {
   return rowOfTokens.map(token => token.value).join('');
+}
+
+export function getTreeStyleLanguages() {
+  return ["C#", "Go", "Java", "JavaScript", "Python", "Rust", "Swift"];
```

Comment:
May need a mapping here, because to call our service, you put the language in the path, but these are the values we support: `android, clang, cpp, dotnet, golang, ios, java, python, rust, typescript` 

So it looks like it will work for Python, Rust and Java, but not the others without a mapping.